app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -87,6 +91,11 @@ const StatusContent = React.createClass({
     this.setState({ hidden: !this.state.hidden });
   },
 
+  handleCollapsedClick (e) {
+    e.preventDefault();
+    this.setState({ collapsed: !this.state.collapsed });
```

Comment:
It flips the boolean. If collapsed is null, it will set collapsed to true as null is falsey.
However, as `handleCollapsedClick` only is used when collapsed is not null that case wont happen.